zipkin/src/test/java/com/linecorp/armeria/client/tracing/HttpTracingClientTest.java

```diff
@@ -64,24 +63,24 @@ public void shouldSubmitSpanWhenSampled() throws Exception {
 
         // check span name
         Span span = reporter.spans().take();
-        assertThat(span.name).isEqualTo(TEST_SPAN);
+        assertThat(span.name()).isEqualTo(TEST_SPAN);
 
         // only one span should be submitted
         assertThat(reporter.spans().poll(1, TimeUnit.SECONDS)).isNull();
 
-        // check # of annotations
-        List<Annotation> annotations = span.annotations;
-        assertThat(annotations).hasSize(2);
+        // check # of annotations (zipkin2 format does not use them by default)
+        assertThat(span.annotations()).isEmpty();
 
-        // check annotation values
-        List<String> values = annotations.stream().map(anno -> anno.value).collect(Collectors.toList());
-        assertThat(values).containsExactlyInAnyOrder("cs", "cr");
+        // check tags
+        assertThat(span.tags()).containsAllEntriesOf(ImmutableMap.of(
+                "http.host", "localhost",
+                "http.method", "POST",
+                "http.path", "/",
+                "http.status_code", "-1",
```

Comment:
maybe better to not put the status code tag instead of adding an invalid one?

Reply:
Yeah in practice it wouldn't happen it was a bad mocking in the test. Cleaned it up and some inconsistencies too.